README.md

```diff
@@ -13,6 +13,9 @@ A curated list of awesome things related to <a href='https://github.com/marimo-t
 <a href='https://github.com/marimo-team/awesome-marimo'>
 <img src='https://awesome.re/badge-flat2.svg' alt='Awesome'>
 </a>
+<a href="https://doi.org/10.5281/zenodo.12735329">
+<img src="https://zenodo.org/badge/DOI/10.5281/zenodo.14511438.svg" alt="DOI"/>
```

Comment:
i don't know if this is the right place for this / doesn't make sense. 

what is zenodo? can it go below in one of the other sections?

Reply:
> i don't know if this is the right place for this / doesn't make sense.
> 
> what is zenodo? can it go below in one of the other sections?

Zenodo's relevance can be found in the Citation.CFF PR comments - https://github.com/marimo-team/marimo/pull/1778#issuecomment-2223651299; it's usually relevant and commonly seen to have a badge redirect you to the appropriate zenodo URL.

Understandable if you don't want it as such; I just felt it was relevant (can be found in popular libraries - https://github.com/arviz-devs/arviz, https://github.com/numpy/numpy, etc).

Reply:
Oh I see, sorry I am not from the scientific community so had no clue. I'll defer to @akshayka where he wants that